pkg/api/util.go

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
```

Comment:
Ah. My mistake. I thought you moved them to the utils package, but there just in api/utils.go .Makes sense.